blueprints/popular_genomes.py

```diff
@@ -46,6 +46,9 @@ def get(self, **kwargs):
                 popular_genome['popular_order'] = data['POPULAR_GENOMES'].index(genome_id)
                 popular_genome['is_available'] = False
 
+            popular_genome['svg_name'] = '_'.join(genome_id.split('_')[:2])
```

Comment:
Plain mapping file mapping genome_id to svg would be long term maintainable and sustainable solution here
'genome_id' : 'image_name'
example
'homo_sapiens_GCA_000001405_28' : 'homo_sapiens_38'

Read this mapping in the dictionary and use it wherever is needed. 
This avoid putting the logic for parsing genome_id in order to determine image name. (next two lines) and also assembly checking for grch37 etc.
In future if there is a change in popular species image_name or genome_id it will just need changing the mapping file and no code change at all.